protos/transaction.proto

```diff
@@ -176,5 +182,6 @@ message Transaction {
     Restore restore = 106;
     ReserveFragments reserve_fragments = 107;
     Update update = 108;
+    Project project = 109;
```

Comment:
This is definitely my fault for designing it this way, but it does feel like this can get out of hand...

Reply:
It does carry the risk yeah :D